### ISSUE TITLE
h5py installation on windows - wsl only works when installing via pip

### ISSUE THREAD

**Issue by kenfus** [i4Ds/Karabo-Pipeline#145]
When installing it via the `environment-dev.yaml`, I get the following error when importing h5py:

```
>>> import h5py
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/kenfus/miniconda3/envs/karabo_dev_env/lib/python3.9/site-packages/h5py/__init__.py", line 33, in <module>
    from . import version
  File "/home/kenfus/miniconda3/envs/karabo_dev_env/lib/python3.9/site-packages/h5py/version.py", line 15, in <module>
    from . import h5 as _h5
  File "h5py/h5.pyx", line 1, in init h5py.h5
ImportError: /home/kenfus/miniconda3/envs/karabo_dev_env/lib/python3.9/site-packages/h5py/defs.cpython-39-x86_64-linux-gnu.so: undefined symbol: H5Pget_fapl_direct
```

If I install the version of pip, it works and when I replace the installed version from pip with the one from conda, I get the same error again.

Someone had a similar error: https://github.com/h5py/h5py/issues/1880  and they say that h5py on anaconda was not compiled correctly.

**Comment by kenfus** [comment]
`pip install h5py --force-reinstall` fixes this problem for me.

**Comment by kenfus** [comment]
This seems to be fixed when installing GSL, thus will be fixed with #132
